### ISSUE TITLE
We should recall `api` package back to `volcano` repo

### ISSUE THREAD

**Issue by zen-xu** [volcano-sh/volcano#1460]
### Why is this needed:
We have separated the `api` package from the main repo. #1401

**But it will bring some difficulties to development**

e.g.
If I want to add some experiment fields to `api`, I must submit a pr to `api` repo first. 
But an experiment idea shouldn't be merged into the main thread before it is mature.

### What would you like to be added:
Maybe we can still keep `api` package in `volcano` repo and sync this sub package to its own repo ([like what Kubernetes did](https://github.com/kubernetes/kubernetes/tree/master/staging))

**Comment by shinytang6** [comment]
+1 for this.

**Comment by k82cn** [comment]
> But an experiment idea shouldn't be merged into the main thread before it is mature.

That should be ok for alpha, beta and GA feature.

**Comment by hwdef** [comment]
+1

**Comment by Thor-wl** [comment]
@hwdef Can you help for that?

**Comment by hwdef** [comment]
> @hwdef Can you help for that?

Yes, I can work on it. But a robot may be needed to synchronize the two repo, I don’t know how to deal with it

**Comment by k82cn** [comment]
> Maybe we can still keep api package in volcano repo and sync this sub package to its own repo (like what Kubernetes did)

That's better to me :)

**Comment by hwdef** [comment]
I think this is still needed

**Comment by hwdef** [comment]
@Thor-wl

**Comment by hwdef** [comment]
still need

**Comment by wangyang0616** [comment]
still need

**Comment by kevin-wangzefeng** [comment]
Can we include this in the next release cycle?
We can have `staging/` folder under the v/v and sync content to v/api

**Comment by hwdef** [comment]
> Can we include this in the next release cycle? We can have `staging/` folder under the v/v and sync content to v/api

we need a robot to sync v/v/staging to v/api.

**Comment by hwdef** [comment]
still need
